optimum/commands/export/openvino.py

```diff
@@ -70,9 +70,9 @@ def parse_args_openvino(parser: "ArgumentParser"):
     optional_group.add_argument(
         "--weight-format",
         type=str,
-        choices=["fp32", "fp16", "int8", "int4", "int4_sym_g128", "int4_asym_g128", "int4_sym_g64", "int4_asym_g64"],
+        choices=["fp32", "fp16", "int8", "int4", "mxfp4_e2m1"],
```

Comment:
if we're gonna do this, why not just call it fp4 instead of its full OCP MX name.

Reply:
After some discussion we decided to name specifically `"mxfp4_e2m1"`. This should coney the meaning that the format is the same as in MX specification.

Reply:
> if we're gonna do this, why not just call it fp4 instead of its full OCP MX name.

agreed why not use fp4 or mxfp4 and specify any additional information in the documentation ?

Reply:
@AlexKoff88 @MaximProshin Could you please comment on that?

Reply:
Too many concerns about naming. How about `mxfp4`? This name reflects the format to some degree but there is still an uncertainty in the mantissa/exponent sizes. I think it is ok for now as we support only one format in OpenVINO and have no plans to extend it in the near term.